src/containers/Articles/Article/CreateArticle.jsx

```diff
@@ -0,0 +1,159 @@
+import React, { Component } from "react";
+import { connect } from "react-redux";
+import { EditorState } from "draft-js";
+import { Redirect } from "react-router-dom";
+
+import * as actions from "../../../store/actions/index";
+import CreateArticleForm from "../../../components/UI/Article/ArticleForm";
+import { successToast } from "../../../components/Toast/toast";
+
+export class CreateArticle extends Component {
+
+    state = {
+        editorState: EditorState.createEmpty(),
+        articleData: {
+            title: "",
+            description: "",
+            body: "",
+            image: null
+        },
+        error: false,
+        titleTouched: false,
+        descriptionTouched: false,
+        bodyTouched: false,
+        disabled: false,
+        picture: null
+    };
+
+    onChangeHandler = (editorState) => {
+        this.setState({editorState})
+    };
+
+    bodyChangedHandler=( event, editor ) => {
+        const data = editor.getData();
+        const currentState = {...this.state};
+        const articleData = {...currentState.articleData};
```

Comment:
We would have extracted the `body` and `title` with: 
`const {body, title} = ...currentState.articleData`